modules/connectivity/locals.tf

```diff
@@ -653,11 +655,13 @@ locals {
       resource_id       = local.vpn_gateway_resource_id[location]
       managed_by_module = local.deploy_virtual_network_gateway_vpn[location]
       # Resource definition attributes
-      name                = local.vpn_gateway_name[location]
-      resource_group_name = local.resource_group_names_by_scope_and_location["connectivity"][location]
-      location            = location
-      type                = "Vpn"
-      sku                 = hub_network.config.virtual_network_gateway.config.gateway_sku_vpn
+      name                        = local.vpn_gateway_name[location]
+      resource_group_name         = local.resource_group_names_by_scope_and_location["connectivity"][location]
+      location                    = location
+      type                        = "Vpn"
+      sku                         = hub_network.config.virtual_network_gateway.config.gateway_sku_vpn
+      remote_vnet_traffic_enabled = null
```

Comment:
Should this be `null` or `false`?

Reply:
No, these attributes are only needed for ER deployments of vnet gateway - however two lists of objects are concat'd, the vpn gateways and er gateways. If we do not add null values here, they are dropped in the ER object as all list members must have identical type.